docs/content/en/2.setup.md

```diff
@@ -53,18 +53,63 @@ export default {
 
 See [module options](/api/options).
 
-
 ## TypeScript
 
-Add the types to your "types" array in `tsconfig.json` after the `@nuxt/types` (Nuxt 2.9.0+) or `@nuxt/vue-app` entry.
+If you're using Typescript, add the types to your "types" array in `tsconfig.json` after the `@nuxt/types` (Nuxt 2.9.0+) or `@nuxt/vue-app` entry.
 
 ```json [tsconfig.json]
 {
   "compilerOptions": {
-    "types": [
-      "@nuxt/types",
-      "@nuxt/image"
-    ]
+    "types": ["@nuxt/types", "@nuxt/image"]
   }
 }
 ```
+
+## Troubleshooting
+
+<alert type="info">
+  We recommend using Node LTS version or 12, 14 or 16
+</alert>
+
+To transform your images, Nuxt Image includes `sharp` as a dependency. If a node-gyp error occurs on installation:
+
+- Your architecture is not supported by the pre-built binaries (e.g Apple M1). This means that npm has to build them from source. On Apple systems, this requires python and XCode, on Linux systems python and gcc (`build-essentials`) and on windows, you can check installing requirements with node installer`
```

Comment:
```suggestion
- Your architecture is not supported by the pre-built binaries (for example, this sometimes occurs on Apple M1). This means that npm has to build them from source. On Apple systems, this will require python and XCode, on Linux systems, python and gcc (`build-essentials`) and on Windows, you can check your installation requirements with the node installer.
```

Not sure what that last sentence means. cc: @pi0?